{"uid":"bf7cda02509476b","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"9bbc2729137fa409b9bc8067987e1d0d","time":{"start":1705454108467,"stop":1705454118720,"duration":10253},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55aeee8f4f83 <unknown>\n#1 0x55aeee5adcf7 <unknown>\n#2 0x55aeee5fd99e <unknown>\n#3 0x55aeee5fdaa1 <unknown>\n#4 0x55aeee648d64 <unknown>\n#5 0x55aeee6270dd <unknown>\n#6 0x55aeee646006 <unknown>\n#7 0x55aeee626e53 <unknown>\n#8 0x55aeee5eedd4 <unknown>\n#9 0x55aeee5f01de <unknown>\n#10 0x55aeee8b9531 <unknown>\n#11 0x55aeee8bd455 <unknown>\n#12 0x55aeee8a5f55 <unknown>\n#13 0x55aeee8be0ef <unknown>\n#14 0x55aeee88999f <unknown>\n#15 0x55aeee8e2008 <unknown>\n#16 0x55aeee8e21d7 <unknown>\n#17 0x55aeee8f4124 <unknown>\n#18 0x7f538e694ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f35b54e4bc0>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...iteria', non_existing_project='Non Existing', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"4e3bac02976143f1eeebbe2647780ba2\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f35b558df80>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55aeee8f4f83 <unknown>\nE       #1 0x55aeee5adcf7 <unknown>\nE       #2 0x55aeee5fd99e <unknown>\nE       #3 0x55aeee5fdaa1 <unknown>\nE       #4 0x55aeee648d64 <unknown>\nE       #5 0x55aeee6270dd <unknown>\nE       #6 0x55aeee646006 <unknown>\nE       #7 0x55aeee626e53 <unknown>\nE       #8 0x55aeee5eedd4 <unknown>\nE       #9 0x55aeee5f01de <unknown>\nE       #10 0x55aeee8b9531 <unknown>\nE       #11 0x55aeee8bd455 <unknown>\nE       #12 0x55aeee8a5f55 <unknown>\nE       #13 0x55aeee8be0ef <unknown>\nE       #14 0x55aeee88999f <unknown>\nE       #15 0x55aeee8e2008 <unknown>\nE       #16 0x55aeee8e21d7 <unknown>\nE       #17 0x55aeee8f4124 <unknown>\nE       #18 0x7f538e694ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1705454060148,"stop":1705454060148,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1705454060149,"stop":1705454060149,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1705454060149,"stop":1705454060150,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55aeee8f4f83 <unknown>\n#1 0x55aeee5adcf7 <unknown>\n#2 0x55aeee5fd99e <unknown>\n#3 0x55aeee5fdaa1 <unknown>\n#4 0x55aeee648d64 <unknown>\n#5 0x55aeee6270dd <unknown>\n#6 0x55aeee646006 <unknown>\n#7 0x55aeee626e53 <unknown>\n#8 0x55aeee5eedd4 <unknown>\n#9 0x55aeee5f01de <unknown>\n#10 0x55aeee8b9531 <unknown>\n#11 0x55aeee8bd455 <unknown>\n#12 0x55aeee8a5f55 <unknown>\n#13 0x55aeee8be0ef <unknown>\n#14 0x55aeee88999f <unknown>\n#15 0x55aeee8e2008 <unknown>\n#16 0x55aeee8e21d7 <unknown>\n#17 0x55aeee8f4124 <unknown>\n#18 0x7f538e694ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7f35b54e4bc0>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...iteria', non_existing_project='Non Existing', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"4e3bac02976143f1eeebbe2647780ba2\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f35b558df80>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55aeee8f4f83 <unknown>\nE       #1 0x55aeee5adcf7 <unknown>\nE       #2 0x55aeee5fd99e <unknown>\nE       #3 0x55aeee5fdaa1 <unknown>\nE       #4 0x55aeee648d64 <unknown>\nE       #5 0x55aeee6270dd <unknown>\nE       #6 0x55aeee646006 <unknown>\nE       #7 0x55aeee626e53 <unknown>\nE       #8 0x55aeee5eedd4 <unknown>\nE       #9 0x55aeee5f01de <unknown>\nE       #10 0x55aeee8b9531 <unknown>\nE       #11 0x55aeee8bd455 <unknown>\nE       #12 0x55aeee8a5f55 <unknown>\nE       #13 0x55aeee8be0ef <unknown>\nE       #14 0x55aeee88999f <unknown>\nE       #15 0x55aeee8e2008 <unknown>\nE       #16 0x55aeee8e21d7 <unknown>\nE       #17 0x55aeee8f4124 <unknown>\nE       #18 0x7f538e694ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1705454108467,"stop":1705454118720,"duration":10253},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55aeee8f4f83 <unknown>\n#1 0x55aeee5adcf7 <unknown>\n#2 0x55aeee5fd99e <unknown>\n#3 0x55aeee5fdaa1 <unknown>\n#4 0x55aeee648d64 <unknown>\n#5 0x55aeee6270dd <unknown>\n#6 0x55aeee646006 <unknown>\n#7 0x55aeee626e53 <unknown>\n#8 0x55aeee5eedd4 <unknown>\n#9 0x55aeee5f01de <unknown>\n#10 0x55aeee8b9531 <unknown>\n#11 0x55aeee8bd455 <unknown>\n#12 0x55aeee8a5f55 <unknown>\n#13 0x55aeee8be0ef <unknown>\n#14 0x55aeee88999f <unknown>\n#15 0x55aeee8e2008 <unknown>\n#16 0x55aeee8e21d7 <unknown>\n#17 0x55aeee8f4124 <unknown>\n#18 0x7f538e694ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"a5a310a06456e38b","name":"public ip address","source":"a5a310a06456e38b.txt","type":"text/plain","size":11},{"uid":"bfee41273d6fd3ab","name":"Cookies","source":"bfee41273d6fd3ab.json","type":"application/json","size":3872},{"uid":"2364665951551700","name":"Session Storage","source":"2364665951551700.json","type":"application/json","size":359},{"uid":"ff3a10c737a27b9b","name":"Local Storage","source":"ff3a10c737a27b9b.json","type":"application/json","size":208},{"uid":"e3671716e521beb5","name":"Console Logs","source":"e3671716e521beb5.json","type":"application/json","size":2},{"uid":"327eb973454c73f5","name":"Full Page Screenshot","source":"327eb973454c73f5.png","type":"image/png","size":5705799},{"uid":"c7cf466812cc8597","name":"URL","source":"c7cf466812cc8597.uri","type":"text/uri-list","size":23},{"uid":"5b77c5d0cef99f3a","name":"Network Logs","source":"5b77c5d0cef99f3a.json","type":"application/json","size":23114}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"blocker"},{"name":"epic","value":"Security"},{"name":"feature","value":"Login"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az702-226"},{"name":"thread","value":"2317-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'elias@gmail.com'"},{"name":"password","value":"'12345Tr'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"bf7cda02509476b.json","parameterValues":["'elias@gmail.com'","'12345Tr'"]}